netplan/cli/commands/apply.py

```diff
@@ -245,68 +255,49 @@ def is_composite_member(composites, phy):  # pragma: nocover (covered in autopkg
     @staticmethod
     def process_link_changes(interfaces, config_manager):  # pragma: nocover (covered in autopkgtest)
```

Comment:
Some cleanup: after the recent changes, I think we can drop the `interfaces` parameter, as it doesn't seem to be used anymore.

Reply:
Thanks for the review! The `interfaces` parameter is still used ([here](https://github.com/CanonicalLtd/netplan/pull/174/files#diff-40d0f1535a830c32c1e6884b3ce81254a5e9576c1443e059c74f1f4a89d55179R283)). It is passed to `find_matching_iface()` for matching the name glob onto the available network interfaces, so I'll keep that.